strings/artist_strings/names/art_weapon_first.txt

```diff
@@ -80,4 +75,11 @@ Fascist
 Capitalist
 United
 Federated
-Republican
\ No newline at end of file
+Republican
+Modded
+Transformed
+Arcing
+Synthetic
+Valient
```

Comment:
Grammar police: this should be spelled Valiant

Reply:
Fixed